src/main/io/vtx_smartaudio.c

```diff
@@ -922,6 +922,20 @@ static void vtxSASetPitMode(vtxDevice_t *vtxDevice, uint8_t onoff)
         return;
     }
 
+    uint8_t tx805PitMode = vtxSettingsConfig()->tx805PitMode;
+
+    if (tx805PitMode) {
+        // The TX805 is a SmartAudio 2.0 VTX but it interprets the pit mode flags incorrectly.
```

Comment:
In this case the hardware is broken and this needs to be raised with the supplier to get a firmware update or replacement hardware.